### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by anze3db**
Testing some cool new features coming! 🌟

```
1 + 1 = 2
```

# Still a lot to do

* Comments
* Review requested/review dismissed messages
* Commits pushed?
* etc.

But slowly and surely getting there.

### CODE REVIEW

**Review by smotko-mock — CHANGES_REQUESTED** [review]
This could really use some more changes. Don't you think?

**Review by smotko-mock — APPROVED** [review]
On second thought... It's probably fine.

**Review by smotko-mock — COMMENTED** [review]
Or maybe I need to add a comment.

**Review by smotko-mock — APPROVED** [review]
Kk

**Comment by zidarsk8** [comment]
what

**Review by zidarsk8 — DISMISSED** [review]
I think the "cool" should be removed /joke

**Review by smotko-mock — CHANGES_REQUESTED** [review]
I agree, not cool. Not even working apparently.

File: README.md [hunk 1]
```diff
@@ -13,3 +13,5 @@ $ docker-compose exec psywerx bash # You are now in the psywerx container
 # python manage.py syncdb # set up the database tables
 # python manage.py runserver 0.0.0.0:8000 # start the server
 ```
+
+Testing some cool new features. Ignore me :)
```
Comment (zidarsk8) [inline README.md:17]: You shall never be ignored!

**Review by zidarsk8 — CHANGES_REQUESTED** [review]
The last part does not compute. Please remove

**Pull request closed without merging**